client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -58,38 +74,93 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 	}
 
 	onScriptLoaded = ( error ) => {
+		const {
+			isPlaying,
+			item,
+			onScriptLoadError,
+		} = this.props;
+
 		if ( error ) {
-			log( `Script${ error.src } failed to load.` );
+			log( `Script${ get( error, 'src', '' ) } failed to load.` );
+			onScriptLoadError();
+
 			return;
 		}
 
 		const {
-			isPlaying,
-			item,
-		} = this.props;
+			height = 480,
+			videopress_guid,
+			width = 854,
+		} = item;
+
+		if ( ! videopress_guid ) {
+			return;
+		}
 
 		if ( typeof window !== 'undefined' && window.videopress ) {
-			this.player = window.videopress( item.videopress_guid, this.video, {
+			this.player = window.videopress( videopress_guid, this.video, {
 				autoPlay: isPlaying,
-				height: item.height,
-				width: item.width,
+				height: height,
+				width: width,
 			} );
 		}
 	};
 
+	receiveMessage = ( event ) => {
+		if ( event.origin && event.origin !== location.origin ) {
+			return;
+		}
+
+		const data = event.data;
+
+		if ( ! data || 'videopress_loading_state' !== data.event || ! ( 'state' in data ) || ! ( 'converting' in data ) ) {
+			return;
+		}
+
+		if ( ( 'loaded' === data.state ) && ! data.converting ) {
```

Comment:
Since the coding guidelines don't expressly prohibit the use of yoda conditions (and I think an argument could even be made that it's implied that they should be used), I'm going to leave these as is.